packages/field/test-suites/FormatMixin.suite.js

```diff
@@ -344,9 +344,13 @@ export function runFormatMixinSuite(customConfig) {
         // that set hasError back to false when the user input is mimicked.
 
         const AlwaysInvalid = class extends Validator {
+          // eslint-disable-next-line no-useless-constructor
           constructor(...args) {
```

Comment:
eslint is right... let's just remove the constructor here?

Reply:
👌